CHANGELOG.md

```diff
@@ -7,4 +7,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Example use case of spec-generated Ruby client. 
-[cybearl/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
+  [cybearl/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
```

Comment:
```suggestion
  [cyberark/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
```

Reply:
Already fixed in https://github.com/cyberark/conjur-openapi-spec/pull/57/files